src/lib/libp2p.js

```diff
@@ -0,0 +1,74 @@
+import Libp2p from 'libp2p'
+import WS from 'libp2p-websockets'
+import WebRTCStar from 'libp2p-webrtc-star'
+import MPLEX from 'libp2p-mplex'
+import Bootstrap from 'libp2p-bootstrap'
+import { NOISE } from 'libp2p-noise'
+import GossipSub from 'libp2p-gossipsub'
+import { FaultTolerance } from 'libp2p/src/transport-manager'
+
+export const libp2pBundle = (opts) => {
+  const peerId = opts.peerId
+  const bootstrapList = opts.config.Bootstrap
+  return new Libp2p({
+    peerId,
+    addresses: {
+      listen: [
+        /*
+        '/dns4/wrtc-star1.par.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
+        '/dns4/wrtc-star2.sjc.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
+        '/dns4/wrtc-star.discovery.libp2p.io/tcp/443/wss/p2p-webrtc-star',
+        */
+        '/dns4/invalid-wrtc-star.discovery.libp2p.io/tcp/443/wss/p2p-webrtc-star'
+      ]
+    },
+    dialer: {
+      maxParallelDials: 150, // 150 total parallel multiaddr dials
+      maxDialsPerPeer: 4, // Allow 4 multiaddrs to be dialed per peer in parallel
+      dialTimeout: 10e3 // 10 second dial timeout per peer dial
+    },
+    modules: {
+      transport: [WS, WebRTCStar],
+      streamMuxer: [MPLEX],
+      connEncryption: [NOISE],
+      peerDiscovery: [Bootstrap],
+      pubsub: GossipSub
+    },
+    transportManager: {
+      // https://github.com/libp2p/js-libp2p/blob/0a6bc0d1013dfd80ab600e8f74c1544b433ece29/doc/CONFIGURATION.md#configuring-transport-manager
+      // We don't want js-ipfs boot to fail when all webrtc signaling servers are down
+      faultTolerance: FaultTolerance.NO_FATAL
```

Comment:
@vasco-santos @achingbrain  I've refactored things so a custom libp2p bundle is used, and I am still unable to get `FaultTolerance.NO_FATAL` to work when no webrtc server is available. 

Mind eyeballing if I missed something obvious?

Reply:
This is a bug, just created a PR. Thanks @lidel

Reply:
Tested https://github.com/libp2p/js-libp2p/pull/893 and works as expected. 

Were there any breaking changes since last release? 
Do we need to wait for this to bubble up to js-ipfs, or can we bump js-libp2p as an explicit dependency here (when fix is released)?

Reply:
just released it as a patch in libp2p. You should just need to reinstall the dependencies